hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/HotRestartBackupMessageTask.java

```diff
@@ -0,0 +1,91 @@
+package com.hazelcast.client.impl.protocol.task.management;
+
+import com.hazelcast.client.impl.protocol.ClientMessage;
+import com.hazelcast.client.impl.protocol.codec.MCTriggerHotRestartBackupCodec;
+import com.hazelcast.client.impl.protocol.task.AbstractInvocationMessageTask;
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.internal.management.ManagementCenterService;
+import com.hazelcast.internal.management.operation.AbstractManagementOperation;
+import com.hazelcast.internal.nio.Connection;
+import com.hazelcast.spi.impl.operationservice.InvocationBuilder;
+import com.hazelcast.spi.impl.operationservice.Operation;
+
+import java.security.Permission;
+
+import static com.hazelcast.internal.management.ManagementDataSerializerHook.HOT_RESTART_BACKUP_OPERATION;
+
+class HotRestartBackupOperation extends AbstractManagementOperation {
+
+    private final Node node;
+
+    HotRestartBackupOperation(Node node) {
+        this.node = node;
+    }
+
+    @Override
+    public void run()
+            throws Exception {
+        node.getNodeExtension().getHotRestartService().backup();
```

Comment:
Why do we need to have this in a dedicated operation? AFAICT it's enough to call this method on a single member, which would then act as a coordinator and trigger hot backup on all members.

Reply:
Fixed, thanks.